frontend/pages/beta/index.tsx

```diff
@@ -17,44 +17,32 @@ import {
 } from '@mui/material/'
 import { useTheme } from '@mui/material/styles'
 import Link from 'next/link'
-import React, { Fragment, useEffect, useState } from 'react'
+import React, { Fragment, useState } from 'react'
 
 import { useListModels } from '../../actions/model'
-import { ListModelType } from '../../data/model'
 import ChipSelector from '../../src/common/ChipSelector'
 import EmptyBlob from '../../src/common/EmptyBlob'
 import MultipleErrorWrapper from '../../src/errors/MultipleErrorWrapper'
 import Wrapper from '../../src/Wrapper.beta'
-import { MarketPlaceModelGroup, MarketPlaceModelSelectType } from '../../types/types'
 import useDebounce from '../../utils/hooks/useDebounce'
 
 export default function ExploreModels() {
-  const [group, setGroup] = useState<ListModelType>('all')
   // TODO - fetch model tags from API
   const [filter, setFilter] = useState('')
-  const [selectedLibrary, setSelectedLibrary] = useState('')
+  const [selectedLibraries, setSelectedLibraries] = useState<string[]>([])
   const [selectedTask, setSelectedTask] = useState('')
-  const [selectedType, setSelectedType] = useState<MarketPlaceModelSelectType | ''>('')
+  const [selectedTypes, setSelectedTypes] = useState<string[]>([])
   const debouncedFilter = useDebounce(filter, 250)
 
-  const { models, isModelsError, mutateModels } = useListModels(group, debouncedFilter)
+  const { models, isModelsError } = useListModels(
+    selectedTypes.join(),
```

Comment:
Why do you have `.join()` here.  This will turn it into a string like `JAX,Transformers` which isn't how the search works.

Reply:
You can either do:

`list=a,b,c,d` **or** `list=a&list=b&list=c&=d` I believe both are valid depending how the backend is set up, I wasn't sure which way we were doing it so made a guess. I'll change it to the latter approach now, though!